ligo/stablecoin/actions.ligo

```diff
@@ -141,7 +141,9 @@ function debit_from
       { const updated_balance : nat = abs (ledger_balance - parameter.amount)
       ; updated_ledger := Big_map.update(parameter.from_, Some (updated_balance), updated_ledger)
       }
-  | None -> failwith ("NOT_IN_LEDGER")
+  | None -> if parameter.amount > 0n
+      then failwith ("INSUFFICIENT_BALANCE")
```

Comment:
Isn't it more efficient to assign `current_balance = case store.ledger[parameter.from_] of Some balance -> balance; None -> 0` and then apply debit logic regardless of the amount? I suppose in this case there will be less instruction or maybe about the same amount of instructions, but a relatively long string (`INSUFFICIENT_BALANCE`) will appear only once. Also it will more clearly correspond to the following FA2 statement:
> If the holder does not hold any tokens of type token_id, the holder's balance is interpreted as zero.

Reply:
Yes, right. Changed.